tests/test_jobs.py

```diff
@@ -146,13 +145,15 @@ def test_create_spark_job(client, mocker, notebook_maker,
         size=spark_job.size,
         user_email=test_user.email,
     )
+    spark_job.latest_run = spark_job.get_latest_run()
```

Comment:
Because `spark_job.run()` creates a new latest run that needed to be forcefully updated on the `spark_job` object. This will be fixed by doing what you say to do above, which is to invalidate the old `latest_run`. I like that better.